apps/framework-docs/src/pages/building.mdx

```diff
@@ -0,0 +1,29 @@
+import { Cards, Card } from 'nextra/components'
+
+# Building a Data Service
+MooseJS helps you build data intensive applications and service. In its current state, MooseJS can't be used to build a full fledged web application. However, it can be used to build a data service which you can use to power adhoc analytics use cases.
```

Comment:
I am not sure we should talk about what it can't do yet. ie I would remove: `MooseJS can't be used to build a full fledged web application` It would make it more positive.